test/instance.ts

```diff
@@ -450,9 +462,7 @@ describe('Instance', () => {
     });
 
     it('should return true if error is absent', done => {
-      instance.getMetadata = callback => {
-        callback(null);
-      };
+      sandbox.stub(instance, 'getMetadata').yields(null);
```

Comment:
Should this be `resolves()`?

Reply:
here `resolves()` fails this test case..